build.gradle

```diff
@@ -60,7 +60,7 @@ repositories {
 
 dependencies {
     implementation 'log4j:log4j:1.2.17',
```

Comment:
Let's remove log4j as an `implementation` dependency - we are using slf4j, which allows users to choose their logger implementation - make sure to add it as a `testImplementation` dependency though, as we do use it directly in the unit tests.

Reply:
done